pkg/web/router/router.go

```diff
@@ -70,38 +70,65 @@ func SetAppRoutes(r *echo.Group, appsHandler apps.HTTPHandler) {
 	//     description: App not found
 	r.GET("/apps/{id}", appsHandler.GetAppByID) // TODO: Implement correctly the call of the method passing the parameters
 
-	// swagger:operation PUT /apps/{id} App
+	// swagger:operation PUT /apps/:id/versions Version
```

Comment:
@davidffrench note that the id here is not required to perform the action. Should not it better it be /versions only?